pandas/tseries/tests/test_timedeltas.py

```diff
@@ -944,6 +945,27 @@ def test_fields(self):
 
         tm.assert_series_equal(s.dt.days,Series([1,np.nan],index=[0,1]))
         tm.assert_series_equal(s.dt.seconds,Series([10*3600+11*60+12,np.nan],index=[0,1]))
+        
+    def test_total_seconds(self):
```

Comment:
add a test in ``test_series/test_dt_namespace_accessor`` as well (its like the one you have for test Series)